.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and publish artifacts
+
+on: push
+
+jobs:
+  build-artifacts:
+    name: Build Python packages
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+        
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+
+    - name: Archive build artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: dist
+        path: |
+          dist
+
+  publish-artifacts:
+    name: Publish Python packages
+    runs-on: ubuntu-latest
+    needs: [build-artifacts]
+
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
I am not sure if this is strictly necessary but it doesn't hurt. It seems to not take much if any time:

![image](https://user-images.githubusercontent.com/47774/193701472-c9422b05-1016-4be8-8183-af30c83b3719.png)